plugins/CoreHome/angularjs/report-export/reportexport.popover.html

```diff
@@ -61,7 +61,8 @@
     </div>
 
     <div class="col l12" ng-show="showUrl">
-        <textarea piwik-select-on-focus readonly class="exportFullUrl" title="{{ 'CoreHome_ExportTooltip'|translate }}">{{ getExportLink() }}</textarea>
+        <textarea piwik-select-on-focus readonly class="exportFullUrl">{{ getExportLinkWithoutToken() }}</textarea>
+        <div class="tooltip" ng-bind-html="'CoreHome_ExportTooltipWithLink'|translate:'<a target=_blank href=\'/index.php?module=UsersManager&action=userSecurity\'>':'</a>':'ENTER_YOUR_TOKEN_AUTH_HERE'"></div>
```

Comment:
This URL starting w/ `/index.php` will break on subdirectory installs, correct?

Reply:
good catch, thanks. I'll check it.

Reply:
hi @diosmosis I fixed this, using directory-relative URL instead of origin-relative.